packages/stark-ui/typings/text-mask-core/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// typings not yet provided in the text-mask library
+// TODO: remove these and use the ones from DefinitelyTyped once they are implemented
```

Comment:
Could you add a link to the issue?

Reply:
Yeah, I haven't submitted it yet but for sure I'll do and I'll add this in my next PR for the other mask directives ;)